--- old/nginx-1.22.0/src/event/modules/ngx_epoll_module.c	2022-05-24 07:59:19.000000000 +0800
+++ new/nginx-1.22.0/src/event/modules/ngx_epoll_module.c	2022-06-05 16:04:53.918118766 +0800
@@ -832,7 +832,7 @@
                       "epoll_wait() returned no events without timeout");
         return NGX_ERROR;
     }
-
+    batch_start();
     for (i = 0; i < events; i++) {
         c = event_list[i].data.ptr;
 
@@ -931,7 +931,7 @@
             }
         }
     }
-
+    batch_flush();
     return NGX_OK;
 }
 